core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java

```diff
@@ -261,7 +261,8 @@ private void handleRequest(ChannelHandlerContext ctx, DecodedHttpRequest req) th
         final PathMapped<ServiceConfig> mapped;
         try {
             mapped = host.findServiceConfig(mappingCtx);
-        } catch (HttpResponseException cause) {
+        } catch (HttpStatusException cause) {
+            // We do not use HttpResponseException internally.
```

Comment:
How about:

    // We do not need to handle HttpResponseException here because we do not use it internally.

for clarification?